Modules/IO/MeshBase/include/itkMeshIOBase.h

```diff
@@ -802,9 +802,13 @@ class ITKIOMeshBase_EXPORT MeshIOBase : public LightProcessObject
         auto numberOfPoints = static_cast<unsigned int>(input[inputIndex++]);
         if (numberOfPoints > 2 && cellType == CellGeometryEnum::LINE_CELL)
         {
-          cellType = CellGeometryEnum::POLYLINE_CELL;
+          auto newCellType = CellGeometryEnum::POLYLINE_CELL;
+          output[outputIndex++] = static_cast<TOutput>(newCellType);
```

Comment:
Please consider merging these two lines to one:

    output[outputIndex++] = static_cast<TOutput>(CellGeometryEnum::POLYLINE_CELL);

It does not seem necessary to introduce an extra non-const variable, "newCellType" here.